lib/src/Base/Graph/openturns/DrawableImplementation.hxx

```diff
@@ -82,6 +82,9 @@ public:
   virtual void setColor(const String & color);
   virtual String getEdgeColor() const;
 
+  /** Explicit color validation flag accessor */
+  virtual Bool isColorExplicitlySet() const;
```

Comment:
this should be a protected method as I dont think there is a legitimate use of it outside of graphimplementation
(DrawableImplementation is friend class of GraphImplementation so it will be allowed to call it)